rero/rero-ils
# holdings: display holdings records

* Adds holdings records display to the document detailed view.

Co-Authored-by: Sebastien Deleze <sebastien.deleze@rero.ch>
Co-Authored-by: Aly Badr <aly.badr@rero.ch>
Co-Authored-by: bertrand Zuchuat <bertrand.zuchuat@rero.ch>

To test:
* display a document detailed view and make sure information about holdings and items are displayed correctly
